.github/actions/action-aqua-scan/action.yml

```diff
@@ -77,7 +77,10 @@ runs:
         mkdir -p evaluations
         docker run -v "$PWD/evaluations:/evaluations" "${{ inputs.aqua-image }}" \
         scan --token "${{ inputs.aqua-access-token }}" --host "${{ inputs.aquasec-server-url }}" \
-        --registry "${{ steps.prepare-input.outputs.image-registry }}" --show-negligible --jsonfile /evaluations/evaluation_"${{ steps.prepare-input.outputs.safe-image-id }}".json "${{ steps.prepare-input.outputs.image-name-no-registry }}:${{ steps.prepare-input.outputs.image-tag }}"
+        --registry "${{ steps.prepare-input.outputs.image-registry }}" --show-negligible --jsonfile /evaluations/evaluation_"${{ steps.prepare-input.outputs.safe-image-id }}".json "${{ steps.prepare-input.outputs.image-name-no-registry }}:${{ steps.prepare-input.outputs.image-tag }}" || true
+        if [ ! -f $PWD/evaluations/evaluation_"${{ steps.prepare-input.outputs.safe-image-id }}".json ]; then
+          echo "File with scan output not created" && false
```

Comment:
When this can happen? I am not sure whether the error output can help investigate what went wrong.

Reply:
The docker has it's own output, that should be sufficient to determine what went wrong. I'm not sure about all the possible errors, that could lead to this. I just wanted to make sure that when the docker output is non-zero, (such as image policy failure) it doesn't crash the pipeline, if it produces a valid scan json file.

Reply:
And at the same time, crash fast, if the failure is severe enough, to not  produce a file.

Reply:
Might deserve a comment with this rationale. Otherwise the `|| true` part on the docker run is a bit mysterious.